### PR TITLE
#65 Drop support of old PHP versions

### PR DESCRIPTION
**Pull request by laurent35240**

### CODE REVIEW

**Comment by laurent35240** [comment]
Sure, I added `deps=low` for the build with PHP 7.1
I did not add `deps=high` since it is the default value
I put only one `coverage=true` because I do not see the point of calculating code coverage twice.

BTW I noticed that the Github Access Token became invalid on Scrutinizer
Code Quality Summary - pyrech composer-changelogs - Measure and Improve Code Quality continuously with Scrutinizer

File: .travis.yml [hunk 1]
```diff
@@ -8,15 +8,8 @@ cache:
...
-        - php: 5.4
-          env: check_cs=true
-        - php: 5.5
-        - php: 5.6
-          env: deps=low coverage=true
-        - php: 5.6
-          env: deps=high coverage=true
-        - php: 7.0
         - php: 7.1
+          env: deps=low check_cs=true coverage=true
```
Comment (pyrech) [inline .travis.yml]: I would prefer to not run the check_cs with the lowest dependencies.

**Comment by pyrech** [comment]
I just asked a renew of the Scrutinizer token... I think, not sure it worked. I wonder if I should not drop Scrutinizer completly as it's not really useful actually.
Anyway, I just left one comment, and once fixed, I will merge this PR and release a new minor version \o/

File: .travis.yml (hunk 1 above)
Comment (laurent35240) [reply .travis.yml]: Sure I changed it.
For curiosity, any particular reason for not running code style check with lowest dependencies?

**Comment by laurent35240** [comment]
Maybe we can move from Scrutinizer to coveralls if Scrutinizer was just used for code coverage

**Comment by pyrech** [comment]
Awesome, thank you @laurent35240

**Pull request merged**